src/lib/layout/responsiveContainerHeader.svelte

```diff
@@ -97,14 +104,14 @@
                 {/if}
 
                 {#if showSearch && hasSearch}
-                    <SearchQuery placeholder={searchPlaceholder} />
+                    <SearchQuery bind:this={searchQuery} placeholder={searchPlaceholder} />
```

Comment:
I think we should avoid this pattern. we have many search inputs and creating a binding instance is not the best DX. we should be somehow able to listen to the changes for clear button and clear queries. We already also have -
```ts
queries.clearAll();
```